File:  LeetCode_75/Day1/217_Contains_Duplicate.py
from typing import List

def containsDuplicate(nums: List[int]) -> bool:
        nums_dict = {}
        flag = False
        for num in nums:
            if num in nums_dict:
                flag = True
                break
            nums_dict[num] = 1
        return "true" if flag else "false"

print(containsDuplicate([1,2,3,4]))

File:  LeetCode_75/Day1/217_Contains_Duplicate.cpp
#include <map>
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        map<int, int> nums_map;
        int i, size=nums.size();
        nums_map.insert({ nums[0], 1 });
        for(i=1;i<size;i++) {
            if(nums_map.find(nums) == nums_map.end())
                return true;
            nums_map.insert(nums[i], 1);
        }
        return false;

    }
};


/*
/*
map<string, int> mapset;
int main(void) {

	mapset.insert({ "Alice", 100 });
	mapset.insert({ "Bob", 200 });

	if (mapset.find("Alice") != mapset.end()) 
*/